adagrams/game.py

```diff
@@ -1,11 +1,125 @@
+
+import random
+
+
 def draw_letters():
-    pass
+    # array_of_strings = [ ]
+    LETTER_POOL = {
```

Comment:
This is a great dictionary to hold the letters and the count of each letter! When creating it, you've followed the naming convention of a global variable which I agree would be the best way to hold this information just in case we need to use this dictionary elsewhere! The one change I would make here would be to move it outside this function which is what we would do to make sure the dictionary is available to the other functions that might need it. If we have several pieces of this kind of information, it is also possible to put your data in a separate file and import it in as well!